test/routes/appendix/appendix.test.js

```diff
@@ -0,0 +1,218 @@
+const HTTP_STATUS = require('http-status-codes');
+const supertest = require('supertest');
+const getPort = require('get-port');
+const db = require('../../../app/models');
+
+const CONFIG = require('../../../app/config');
+const {listen} = require('../../../app');
+
+CONFIG.set('env', 'test');
+const {username, password} = CONFIG.get('testing');
+
+let server;
+let request;
+
+const CREATE_DATA = {
+  text: '<h3>Title</h3><p>Test text</p>',
+};
+
+const UPDATE_DATA = {
+  text: '<h2>Updated Title</h2><p>Updated test text</p>',
+};
+
+const templateAppendixProperties = [
+  'ident', 'createdAt', 'updatedAt', 'text',
+];
+
+const checkTemplateAppendix = (appendixObject) => {
+  templateAppendixProperties.forEach((element) => {
+    expect(appendixObject).toHaveProperty(element);
+  });
+  expect(appendixObject).toEqual(expect.not.objectContaining({
+    id: expect.any(Number),
+    templateId: expect.any(Number),
+    updatedBy: expect.any(Number),
+    deletedAt: expect.any(String),
+  }));
+};
+
+// Start API
+beforeAll(async () => {
+  const port = await getPort({port: CONFIG.get('web:port')});
+  server = await listen(port);
+  request = supertest(server);
+});
+
+describe('/appendix', () => {
+  let template;
+  let project;
+  let body;
+
+  beforeAll(async () => {
+    // create a template to be used in tests
+    template = await db.models.template.create({
+      name: 'Test Template',
+      organization: 'Test Org',
+      sections: [
+        'microbial',
+        'msi',
+        'small-mutation',
+      ],
+    });
+
+    // Create project
+    project = await db.models.project.create({name: 'test-appendix-project'});
+
+    body = {
+      templateId: template.id,
+      projectId: project.id,
+    };
+  });
+
+  describe('GET', () => {
+    let getTestAppendix;
+
+    beforeEach(async () => {
```

Comment:
beforeEach and afterEach in each describe block is redundant, setup and teardown could be moved to top level to simplify the code